state/movies/actions.js

```diff
@@ -0,0 +1,19 @@
+import * as types from './types';
+
+export const fetchTrendingMovies = (page) => {
```

Comment:
In case that you already use babel, you can use next:
```
export const fetchTrendingMovies = (page = 1) => {}
```
If you pass page as undefined, it value will be 1.

Reply:
Fixed.